### ISSUE TITLE
Target Frame percentage value

### ISSUE THREAD

**Issue by WrobotProfiles** [obble/modui_classic#68]
seems to over write default blizz show  percentabe values.

and when you turn off show percentages, well modui doesnt show it either =( add option? or take off?

https://i.imgur.com/86CoPNJ.png

**Comment by WrobotProfiles** [comment]
https://i.imgur.com/frO8thu.png

**Comment by petterio** [comment]
im getting the same https://gyazo.com/99c2b38aac2a4127df505e661f1862d7 might be another addon tho

**Comment by Andoido1** [comment]
im 100% sure its modui - when you disable the addon it goes away.
and only way to show % values is through interface options,.

**Comment by obble** [comment]
blizzard doesn't show target values by default. are you using an addon like [RealMobHealth](https://www.wowinterface.com/downloads/info24924-RealMobHealth.html) or [LibClassicMobHealth](https://www.curseforge.com/wow/addons/libclassicmobhealth-1-0) outside of the modui package?

**Comment by petterio** [comment]
yeah ur right. im using realmobhealth. is it possible to incorporate that into modui ?
